PopulateTools/gobierto
# Scopes front

Closes #1002 

### What does this PR do?

Add front to scopes

### How should this be manually tested?

- Yo can create a scope from admin (now, you have the slug attribute)
- In front, you can see in Parcipation Scopes link to see the scopes, and see one scope, then go from processes to scopes by the link that has to be the reference.
- I added several scopes tests that were pending

<img width="1226" alt="captura de pantalla 2018-01-12 a las 7 32 49" src="https://user-images.githubusercontent.com/69764/34862478-e35b9468-f76a-11e7-9ae5-4498420b6107.png">
